scripts/install

```diff
@@ -17,32 +17,31 @@ ynh_script_progression --message="Storing installation settings..." --weight=2
 if [ "$dns_over_https" == "1" ];
 then
     dns_over_https="true"
-    # DNS over TLS
-    adguard_DoT_port=853
-    ynh_app_setting_set --app="$app" --key=adguard_DoT_port --value=$adguard_DoT_port
-    ynh_exec_warn_less yunohost firewall allow --no-upnp UDP $adguard_DoT_port
-    ynh_exec_warn_less yunohost firewall allow --no-upnp TCP $adguard_DoT_port
-    # DNS over QUIC
-    adguard_DoQ_port=784
-    ynh_app_setting_set --app="$app" --key=adguard_DoQ_port --value=$adguard_DoQ_port
-    ynh_exec_warn_less yunohost firewall allow --no-upnp UDP $adguard_DoQ_port
+    # no need to open the ports, as they were opened at the 'Provisioning ports' step
 else
     dns_over_https="false"
+    # if dns_over_https is false, we need to close ports,
+    # as they were opened at the 'Provisioning ports' step
+    ynh_exec_warn_less yunohost firewall disallow --no-upnp TCP "$port_dns_over_http"
+    ynh_exec_warn_less yunohost firewall disallow --no-upnp UDP "$port_dns_over_http"
+    ynh_exec_warn_less yunohost firewall disallow --no-upnp UDP "$port_dns_over_quic"
```

Comment:
```suggestion
    ynh_exec_warn_less yunohost firewall disallow Both "$port_dns_over_http"
    ynh_exec_warn_less yunohost firewall disallow UDP "$port_dns_over_quic"
```

(there's no --no-upnp option apparently ... much confused by the API but UPNP integration is like not maintained since years x_x)